OpenRA.Mods.Cnc/Traits/Render/WithBuildingBib.cs

```diff
@@ -12,18 +12,21 @@
 using System.Collections.Generic;
 using OpenRA.Graphics;
 using OpenRA.Mods.Common.Graphics;
+using OpenRA.Mods.Common.Traits;
 using OpenRA.Mods.Common.Traits.Render;
 using OpenRA.Traits;
 
-namespace OpenRA.Mods.Common.Traits
+namespace OpenRA.Mods.Cnc.Traits
 {
-	public class BibInfo : ITraitInfo, Requires<BuildingInfo>, IRenderActorPreviewSpritesInfo, IActorPreviewInitInfo, Requires<RenderSpritesInfo>
+	public class WithBuildingBibInfo : ITraitInfo, Requires<BuildingInfo>, IRenderActorPreviewSpritesInfo, IActorPreviewInitInfo, Requires<RenderSpritesInfo>
 	{
 		[SequenceReference] public readonly string Sequence = "bib";
+
 		[PaletteReference] public readonly string Palette = TileSet.TerrainPaletteInternalName;
-		public readonly bool HasMinibib = false;
 
-		public object Create(ActorInitializer init) { return new Bib(init.Self, this); }
+		public readonly bool IsMiniBib = false;
```

Comment:
I haven't checked yet, but don't mini-bibs use tiled artwork as well? In that case we'd have to go with the first fix, at least for mini-bibs larger than a single tile.